### PR TITLE
changed version of MbedTLS in .travis.yml

### PR DESCRIPTION
**Pull request by SabineAuer**
this is to test whether with an older version of MbedTLS the travis CI is functioning again

### CODE REVIEW

**Comment by SabineAuer** [comment]
@timkittel: temporarily the travis CI is fixed. Do you want to merge this into the branch master?

File: .travis.yml [hunk 1]
```diff
@@ -19,10 +19,11 @@ env:
...
 
-addons:
-  apt:
-    packages:
-    - MbedTLS
+before_script:
+  - julia -e 'using Pkg;
+      Pkg.add(PackageSpec(name="MbedTLS", version="0.6.6"));
+      Pkg.pin(PackageSpec(name="MbedTLS", version="0.6.6"));
+      Pkg.add([PackageSpec(name="BinaryBuilder", rev="master"), PackageSpec(name="BinaryProvider")])'
```
Comment (timkittel) [inline .travis.yml]: Is the use of the master branch of `BinaryBuilder` here strictly necessary? `MbedTLS` worked before so there should be a working version METADATA. Depending on the master branch of an external repo kinda asks for having an unstable ci.

Also, why do you add `BinaryBuilder` twice? And why is there a transpose oeprator behind the command?

**Comment by timkittel** [comment]
> @timkittel: temporarily the travis CI is fixed. Do you want to merge this into the branch master?

Just left comments (:

File: .travis.yml (hunk 1 above)
Comment (SabineAuer) [reply .travis.yml]: If you think, the `BinaryBuilder` is not necessary, I take it out.
Concerning the "transpose operator". In line 24 there stands: "julia -e 'using..." and it is the start of the command which ends in line 26 with the "transpose operator".

**Comment by timkittel** [comment]
Looks good. Why did you have the BinaryBuilder there in the first place? Merging now.

**Pull request merged**